contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -621,7 +548,6 @@ contract StakeManager is IStakeManager {
     ) internal returns (uint256) {
         // total voting power
         uint256 _stakePower;
-        //TODO: add proposer bonus
```

Comment:
Not sure what "proposer bonus" is, but did we add it, since we are removing the todo?

Reply:
no, as of now choose to not give any bonus to proposers.